internal/generator/vector/elements/match.go

```diff
@@ -0,0 +1,44 @@
+package elements
+
+import (
+	"github.com/openshift/cluster-logging-operator/internal/generator"
+)
+
+type Match struct {
```

Comment:
Can we name this whatever vector calls it.  I don't think its "match'

Reply:
Cleanup the code now.